slave/process-voms-dirac/lib/process-voms_dirac.pl

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/perl
+
+# initialize parser and read the file
+use Sys::Syslog;
+use XML::Simple;
+use Text::CSV;
+use Data::Dumper;
+use Array::Utils qw(:all);
+use JSON::XS;
+use File::Basename;
+use File::Path qw/make_path/;
+my $vos = XMLin( '-',
+	ForceArray => [ 'role', 'group', 'user', 'vo' ],
+#	GroupTags => { role => 'roles', groups => 'group', users => 'user', vos => 'vo' },
+	KeyAttr => [] );
+my $csv = Text::CSV->new({ sep_char => ',' });
+
+$attributeStatusFilePrefix = "/var/lib/perun-services/process-voms-attributes-";
```

Comment:
This path will be: /var/lib/perun/process-voms-dirac/ and there should be the cache file.